modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -821,8 +906,11 @@ class PrePostProcWrapper {
     void cfgPreProcessing(const std::string  &input_name,
                           const cv::GMetaArg &input_meta,
                           const bool         disable_img_resize = false) {
-        GAPI_Assert(cv::util::holds_alternative<cv::GMatDesc>(input_meta));
-        const auto &matdesc = cv::util::get<cv::GMatDesc>(input_meta);
+        GAPI_Assert(cv::util::holds_alternative<cv::GMatDesc>(input_meta) ||
+                    cv::util::holds_alternative<cv::GFrameDesc>(input_meta));
+        auto isMat = cv::util::holds_alternative<cv::GMatDesc>(input_meta);
```

Comment:
If `isImage` handles `GFrameDesc` It can be simplified like this:
```
auto prec = isMat ? cv::util::get<cv::GMatDesc>(input_meta).depth : CV_8U;
// Here is isImage handles GFrameDesc as well
if isImage(input_meta) {
    const auto size = isMat ? cv::util::get<cv::GMatDesc>(input_meta).size
                                           : cv::util::get<cv:GFrameDesc>(input_meta).size
    ...
    if (!disable_img_resize) {
        input_info.tensor().set_spatial_static_shape(size.height, size.width)
    }
    ...
    ss << "..." << ( isMat ? cv::util::get<cv::GMatDesc>(input_meta)
                                       : cv::util::get<cv:GFrameDesc>(input_meta) ) << "..."
    ...
}
```

Reply:
Fixed, thanks! https://github.com/opencv/opencv/pull/24938/commits/833bd890ef003a7c161f3a91c61b9c8d81df0e55